internal/kubernetes/client/client.go

```diff
@@ -125,6 +125,7 @@ func (client Client) GetReplicaSet(name, namespace string) (*appsv1.ReplicaSet,
 func (client Client) GetNode(name string) (*corev1.Node, error) {
 	p, err := client.Clientset.CoreV1().Nodes().Get(context.Background(), name, metav1.GetOptions{})
 	if err != nil {
+		utils.PrintLog("error", utils.LogLine{Message: fmt.Sprintf("%v", err)})
```

Comment:
can you remove this line? the error is returned, it's enough

Reply:
Yes, I'll change the message though because that was misleading, other errors can occur that are not due the fact node doesn't exist.